pymaid/pymaid.py

```diff
@@ -1139,11 +1139,12 @@ def get_connectors ( skids, remote_instance = None, incoming_synapses = True, ou
                         if True, gap junctions will be retrieved
     project_id :        int (default = 1)
                         ID of the CATMAID project
+    tag_flag :          set if tags of connectors should be retrieved. Possible values = 0/false or 1/True

```

Comment:
I would suggest pulling tags as default as it doesn't cost extra time. But make it True/False instead of 0/1 to keep it consistent with other parameters